app/dropdown.tsx

```diff
@@ -0,0 +1,11 @@
+import { View } from 'react-native';
+import DropdownComponent from '../components/features/DropdownComponent/DropdownComponent';
+import HeaderWrapper from '../components/ui/Navigation/HeaderWrapper';
+
+export default function DropdownPage() {
```

Comment:
Instead of making this a page for demo purposes. We could work this into the header wrapper. So instead of having a home button we could have the dropdown with other actions